src/sentry/issues/escalating_issues_alg.py

```diff
@@ -0,0 +1,113 @@
+"""escalating_issues_final_alg.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1bEuyWwwUvmCKiCzo3Nh0HaWzQFnvokPI
+Validation is located at
+    https://github.com/getsentry/data-analysis/blob/main/escalating_issues_analysis/escalating_issues_validation_tool.ipynb
+"""
+
+import math
+import statistics
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Union
+
+ForecastObject = Dict[str, Union[int, str]]
+
+
+def generate_issue_forecast(data: Dict[str, Any], start_time: datetime) -> List[ForecastObject]:
```

Comment:
Any way we can disambiguate whats in `data`?

Reply:
I've changed this to `Dict[str, List[Any]]`, but I cannot disambiguate further than that, as the lists can be of type `int` or `str`. If I try to type this as `Union[str, int]`, typing complains, as the integer comparisons and string conversions are done on `data`.